app/templates/components/ui-table/cell/events/view/tickets/orders/cell-actions.hbs

```diff
@@ -1,8 +1,10 @@
 <div class="ui vertical compact basic buttons">
-  {{#ui-popup content=(t 'Cancel order') class='ui icon button' position='left center'}}
-    <i class="delete icon"></i>
-  {{/ui-popup}}
-  {{#ui-popup content=(t 'Delete order') class='ui icon button' position='left center'}}
+  {{#if (not-eq record.status 'cancelled')}}
+    {{#ui-popup content=(t 'Cancel order') click=(action (confirm (t 'Are you sure you would like to cancel this Order?') (action cancelOrder record))) class='ui icon button' position='left center'}}
+      <i class="delete icon"></i>
+    {{/ui-popup}}
+  {{/if}}
+  {{#ui-popup content=(t 'Delete order') click=(action (confirm (t 'Are you sure you would like to delete this Order?') (action deleteOrder record))) class='ui icon button' position='left center'}}
```

Comment:
A completed paid order cannot be cancelled or deleted since we don't have a refund system. It would be great if you can hide it for that case.